model/websocket_client.go

```diff
@@ -17,6 +17,18 @@ const (
 	PING_TIMEOUT_BUFFER_SECONDS = 5
 )
 
+type msgType int
+
+const (
+	msgTypeJSON msgType = iota + 1
+	msgTypePong
+)
+
+type writeMessage struct {
+	msgType  msgType
```

Comment:
Can this just be named `type`? Mainly to avoid the `msg.msgType` redundancy.

Reply:
`type` is a keyword :P

Reply:
`msgType` it is then :sweat_smile: